src/services/run-tests.ts

```diff
@@ -97,7 +97,8 @@ export const runTestsParallel = async (
     ? [sanitizePath(dir)]
     : await listFiles(testDir, configs);
   const filesByConcurrency: string[][] = [];
-  const concurrencyLimit = configs?.concurrency ?? availableParallelism();
+  const concurrencyLimit =
+    configs?.concurrency ?? Math.max(Math.floor(availableParallelism() / 2), 1);
```

Comment:
@wellwelwel What you think about this idea?

Reply:
> What you think about this idea?

It's a great way to prevent `if-else`. LGTM 🙋🏻‍♂️

Reply:
Glad to hear that. Just to say, I just wanted to clean up the code. There is no need for immediate release or so.